docs/folders.md

```diff
@@ -124,6 +125,17 @@ client.folders.delete('12345', {recursive: true}, callback);
 ```
 
 
+Get a Trashed Folder
+---------------
+
+A folder can be retrieved from the trash with the [`folders.getTrashedFolder(folderID, qs, callback)`](http://opensource.box.com/box-node-sdk/Folders.html#getTrashedFolder) method.
```

Comment:
Would you mind clarifying that this does not restore an item from the trash?  I think something like "Information about a folder in the trash can be retrieved with the..." might be easier to understand.